src/bionics.cpp

```diff
@@ -467,7 +467,7 @@ bool player::activate_bionic( int b, bool eff_only )
             vehwindspeed = abs(veh->velocity / 100); // vehicle velocity in mph
         }
         const oter_id &cur_om_ter = overmap_buffer.ter( global_omt_location() );
-        std::string omtername = otermap[cur_om_ter].name;
+        const std::string &omtername = cur_om_ter.obj().name;
```

Comment:
`string_id` allows indirection of the form `cur_om_ter->name`. You could extend `int_id` to include a similar `operator *` to avoid having to write `obj()` everywhere